CITATIONS.md

```diff
@@ -0,0 +1,28 @@
+# nf-core/imcyto: Citations
+
+## Pipeline tools
+
+* [Nextflow](https://www.ncbi.nlm.nih.gov/pubmed/28398311/)
+  > Di Tommaso P, Chatzou M, Floden EW, Barja PP, Palumbo E, Notredame C. Nextflow enables reproducible computational workflows. Nat Biotechnol. 2017 Apr 11;35(4):316-319. doi: 10.1038/nbt.3820. PubMed PMID: 28398311.
+
+* [OpenMS](https://www.ncbi.nlm.nih.gov/pubmed/27575624/)
+  > Röst HL., Sachsenberg T., Aiche S., Bielow C., Weisser H., Aicheler F., Andreotti S., Ehrlich HC., Gutenbrunner P., Kenar E., Liang X., Nahnsen S., Nilse L., Pfeuffer J., Rosenberger G., Rurik M., Schmitt U., Veit J., Walzer M., Wojnar D., Wolski WE., Schilling O., Choudhary JS, Malmström L., Aebersold R., Reinert K., Kohlbacher O. (2016). OpenMS: a flexible open-source software platform for mass spectrometry data analysis. Nature methods, 13(9), 741–748. https://doi.org/10.1038/nmeth.3959. PubMed PMID: 27575624; PubMed Central PMCID: PMC5617107.
+
+* [MSstats](https://www.ncbi.nlm.nih.gov/pubmed/24794931/)
+  > Choi M., Chang CY., Clough T., Broudy D., Killeen T., MacLean B., Vitek O. (2014). MSstats: an R package for statistical analysis of quantitative mass spectrometry-based proteomic experiments. Bioinformatics (Oxford, England), 30(17), 2524–2526. https://doi.org/10.1093/bioinformatics/btu305. PubMed PMID: 24794931.
+
+* [ThermoRawFileParser](https://www.ncbi.nlm.nih.gov/pubmed/31755270/)
+  > Hulstaert N., Shofstahl J., Sachsenberg T., Walzer M., Barsnes H., Martens L., Perez-Riverol, Y. (2020). ThermoRawFileParser: Modular, Scalable, and Cross-Platform RAW File Conversion. Journal of proteome research, 19(1), 537–542. https://doi.org/10.1021/acs.jproteome.9b00328. PubMed PMID: 31755270
+
+## Software packaging/containerisation tools
+
+* [BioContainers](https://www.ncbi.nlm.nih.gov/pubmed/28379341/)
+  > da Veiga Leprevost F, Grüning BA, Alves Aflitos S, Röst HL, Uszkoreit J, Barsnes H, Vaudel M, Moreno P, Gatto L, Weber J, Bai M, Jimenez RC, Sachsenberg T, Pfeuffer J, Vera Alvarez R, Griss J, Nesvizhskii AI, Perez-Riverol Y. BioContainers: an open-source and community-driven framework for software standardization. Bioinformatics. 2017 Aug 15;33(16):2580-2582. doi: 10.1093/bioinformatics/btx192. PubMed PMID: 28379341.
+
+* [Singularity](https://www.ncbi.nlm.nih.gov/pubmed/28494014/)
+  > Kurtzer GM, Sochat V, Bauer MW. Singularity: Scientific containers for mobility of compute. PLoS One. 2017 May 11;12(5):e0177459. doi: 10.1371/journal.pone.0177459. eCollection 2017. PubMed PMID: 28494014; PubMed Central PMCID: PMC5426675.
+
+* [Conda](https://www.ncbi.nlm.nih.gov/pubmed/29967506/)
+  > Grüning B., Dale R., Sjödin A., Chapman BA., Rowe J., Tomkins-Tinch CH., Valieris R., Köster J., Bioconda Team (2018). Bioconda: sustainable and comprehensive software distribution for the life sciences. Nature methods, 15(7), 475–476. https://doi.org/10.1038/s41592-018-0046-7. PubMed PMID: 29967506.
+
+* [Docker](https://www.docker.com/)
```

Comment:
https://dl.acm.org/doi/10.5555/2600239.2600241

Maybe this is the corresponding paper?